### PR TITLE
Send message

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -1548,5 +1548,14 @@
...
+    },
+    "sendMessage": {
+        "title": "Confirm external action",
+        "calculatingFees": "Calculating fees",
+        "loading": "Loading...",
+        "address": "Address",
+        "amount": "Amount",
+        "networkFee": "Network fee",
+        "send": "Send"
     }
```
Comment (sertony) [inline src/ko.json:1560]: @kkkhesha can you translate, please?

**Comment by sertony** [comment]
@vetkamak am I right that these strings refer to the new component you are to put into UIKit for Ever Links?
@BigRoma It looks like we are to put this into the shared components package in UIKit.

File: src/ko.json [hunk 2]
```diff
@@ -1542,5 +1542,14 @@
...
         }
+    },
+    "sendMessage": {
+        "title": "Confirm external action",
+        "calculatingFees": "Calculating fees",
+        "loading": "Loading...",
+        "address": "Address",
+        "amount": "Amount",
+        "networkFee": "Network fee",
+        "send": "Send"
```
Comment (sertony) [inline src/ko.json]: @kkkhesha can you translate, please?

File: src/ru.json [hunk 3]
```diff
@@ -1540,5 +1540,14 @@
...
+    },
+    "sendMessage": {
+        "title": "Confirm external action",
+        "calculatingFees": "Calculating fees",
+        "loading": "Loading...",
+        "address": "Address",
+        "amount": "Amount",
+        "networkFee": "Network fee",
+        "send": "Send"
     }
```
Comment (sertony) [inline src/ru.json]: @kkkhesha can you translate, please?

File: src/ko.json [hunk 4]
```diff
@@ -1542,5 +1542,14 @@
             "useDefaultCard": "기본 카드 사용하기",
             "withdrawToCard": "카드로 인출하기"
         }
+    },
+    "sendMessage": {
+        "title": "Confirm external action",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "title": "외부 활동 확인하기",
```

File: src/ko.json [hunk 5]
```diff
@@ -1542,5 +1542,14 @@
...
+        "calculatingFees": "Calculating fees",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "calculatingFees": "수수료를 계산하는 중...",
```

File: src/ko.json [hunk 6]
```diff
@@ -1542,5 +1542,14 @@
...
+        "loading": "Loading...",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "loading": "로딩 중...",
```

File: src/ko.json [hunk 7]
```diff
@@ -1542,5 +1542,14 @@
...
+        "address": "Address",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "address": "주소",
```

File: src/ko.json [hunk 8]
```diff
@@ -1542,5 +1542,14 @@
...
+        "amount": "Amount",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "amount": "액수",
```

File: src/ko.json [hunk 9]
```diff
@@ -1542,5 +1542,14 @@
...
+        "networkFee": "Network fee",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "networkFee": "네트워크 수수료",
```

File: src/ko.json (hunk 2 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
        "send": "보내기"
```

File: src/ru.json [hunk 10]
```diff
@@ -1540,5 +1540,14 @@
             "useDefaultCard": "Использовать карту по умолчанию",
             "withdrawToCard": "Вывести на карту"
         }
+    },
+    "sendMessage": {
+        "title": "Confirm external action",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "title": "Подтвердите внешнее действие",
```

File: src/ru.json [hunk 11]
```diff
@@ -1540,5 +1540,14 @@
...
+        "calculatingFees": "Calculating fees",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "calculatingFees": "Расчёт суммы комиссии",
```

File: src/ru.json [hunk 12]
```diff
@@ -1540,5 +1540,14 @@
...
+        "loading": "Loading...",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "loading": "Загрузка...",
```

File: src/ru.json [hunk 13]
```diff
@@ -1540,5 +1540,14 @@
...
+        "address": "Address",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "address": "Адрес",
```

File: src/ru.json [hunk 14]
```diff
@@ -1540,5 +1540,14 @@
...
+        "amount": "Amount",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "amount": "Сумма",
```

File: src/ru.json [hunk 15]
```diff
@@ -1540,5 +1540,14 @@
...
+        "networkFee": "Network fee",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "networkFee": "Комиссия сети",
```

File: src/ru.json [hunk 16]
```diff
@@ -1540,5 +1540,14 @@
...
+        "send": "Send"
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
        "send": "Отправить"
```

**Review by sertony — APPROVED** [review]

**Pull request merged**